bundles/automation/org.eclipse.smarthome.automation.core/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,25 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Eclipse SmartHome Automation API
+Bundle-SymbolicName: org.eclipse.smarthome.automation.core
+Bundle-Version: 0.8.0.qualifier
+Bundle-Vendor: Eclipse.org/SmartHome
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Import-Package: 
+ org.eclipse.smarthome.automation,
+ org.eclipse.smarthome.automation.handler,
+ org.eclipse.smarthome.automation.parser,
+ org.eclipse.smarthome.automation.provider,
+ org.eclipse.smarthome.automation.script,
+ org.eclipse.smarthome.automation.template,
+ org.eclipse.smarthome.automation.type,
+ org.eclipse.smarthome.automation.util,
+ org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.core.common.registry,
+ org.json,
```

Comment:
This bundle must not have a dependency to org.json, only the parser.json should have that dependency!